app.js

```diff
@@ -46,6 +46,10 @@ app.use('/kc.json', (request, response) => {
   return response.send(kcJSON);
 });
 
+kc.accessDenied = function (request, response) {
```

Comment:
By overriding this that function, won't all "denined" requests return a 401.  we still also need them to be returning a 403 if necessary.

The current behavior of the keycloak-connect module is a bug, and i've filed a JIRA issue for it https://issues.jboss.org/browse/KEYCLOAK-6810

Reply:
I see. So we can close this PR and I will wait for resolving keycloak bug